test/web-platform-tests/to-run.yaml

```diff
@@ -413,6 +417,12 @@ DIR: html/editing/focus
 composed.window.html: [fail, Not implemented]
 focus-01.html: [timeout, scrollIntoView() not implemented]
 focus-02.html: [timeout, scrollIntoView() not implemented]
+inert/inert-does-not-match-disabled-selector.html: [fail, Not implemented]
```

Comment:
Interesting that these got merged before the corresponding spec change... they should have ".tentative" in their name, in theory. Ah well.